dev_requirements.txt

```diff
@@ -0,0 +1,6 @@
+coveralls   # For `make test-coverage`
+pre-commit  # For `make auto-format-code`
+sphinx      # For `make docs-html` and `make docs-clean`
+wheel       # For `make package`
+twine       # For `make publish`
+black       # For pre-commit hooks
```

Comment:
Why were these dependencies broken out into a separate file?

Reply:
Consistency and sensibleness.  It doesn't really make a lot of sense to put development dependencies in "extras_require", that's really meant more for additional features and the like.